app/api/tickets.py

```diff
@@ -70,7 +70,7 @@ def before_create_object(self, data, view_kwargs):
                     {'event_id': data['event']}, "Event does not exist"
                 )
 
-            if data.get('type') == 'paid':
+            if data.get('type') == 'paid' or 'donation':
```

Comment:
`data.get('type') == 'paid' or 'donation'`

`false or 'donation'`

`false or true`

`true`

Reply:
should I implement separate if block for `donation` and `paid` then?

Reply:
No